core/src/main/java/io/questdb/cairo/TableUtils.java

```diff
@@ -378,38 +381,20 @@ public static Function createCursorFunction(
         return function;
     }
 
-    public static void createMatViewDefinitionFile(MemoryMARW mem, MatViewDefinition matViewDefinition) {
-        mem.extend(MV_HEADER_SIZE);
-        mem.jumpTo(MV_HEADER_SIZE);
+    public static void createMatViewDefinition(AppendableBlock mem, MatViewDefinition matViewDefinition) {
+        final short TYPE = 0;
```

Comment:
nit: lower-case should be used for variable names. Alternatively, these constants could be extracted into `static final` fields.